packages/editor/src/components/convert-to-group-buttons/convert-button.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { MenuItem } from '@wordpress/components';
+import { _x } from '@wordpress/i18n';
+import { switchToBlockType } from '@wordpress/blocks';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import { displayShortcut } from '@wordpress/keycodes';
+
+/**
+ * Internal dependencies
+ */
+import { Group, UnGroup } from './icons';
+
+export function ConvertToGroupButton( {
+	onConvertToGroup,
+	onConvertFromGroup,
+	isGroupable = false,
+	isUnGroupable = false,
+} ) {
+	return (
+		<Fragment>
+			{ isGroupable && (
+				<MenuItem
+					className="editor-block-settings-menu__control block-editor-block-settings-menu__control"
+					icon={ Group }
+					onClick={ onConvertToGroup }
+					shortcut={ displayShortcut.primaryAlt( 'g' ) }
+				>
+					{ _x( 'Group', 'Grouping a selection of blocks together within the Editor' ) }
```

Comment:
I don't pretend to be an expert on localization, but I don't know that this needs to be so use-case specific. I _do_ think it could benefit from context as far as disambiguation between the noun and verb forms.

i.e.


```suggestion
					{ _x( 'Group', 'verb' ) }
```